src/app/api/session/active/complete/route.ts

```
@@ -38,21 +25,10 @@ const completeSessionSchema = z.object({
-    const body = await request.json();
-    const validationResult = completeSessionSchema.safeParse(body);
-
-    if (!validationResult.success) {
-      return errorResponse('Invalid completion data', 400, validationResult.error.errors);
-    }
-
-    const { duration, notes, completedAt } = validationResult.data;
+export const POST = createValidatedApiHandler(
+  completeSessionSchema,
+  async (userId, { duration, notes, completedAt }) => {
+    let finalPerformanceData: any = {};
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)